src/modules/home/screens/home-screen.tsx

```diff
@@ -1,7 +1,145 @@
-import { Text, View } from '../../../ui/core'
+import { Text, View, Image, StyleSheet, ScrollView } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import CarouselComponent from './carousel-component';
+import { ScreenWidth } from '@rneui/base';
+import PaymentMethods from './payment-methods';
 
 export const HomeScreen = () => {
-    return (<View>
-        <Text>Home Screen</Text>
-    </View>)
-}
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <View>
+          <Image
+            style={styles.logo}
+            source={require('../../../ui/assets/images/logo-black-market/Header.png')}></Image>
```

Comment:
You can add the images to the index so you can use them like the others there: 
example `images.backgroundAuth()`

<img width="762" alt="image" src="https://github.com/JUNIOR140889/Black-Market-RN/assets/8755889/7e7b3365-a19d-4690-9ef8-4006e5780f83">

Reply:
done.